packages/amplication-server/src/core/moduleAction/moduleAction.service.ts

```diff
@@ -175,6 +222,19 @@ export class ModuleActionService extends BlockTypeService<
         "Cannot delete a default Action for entity. To delete it, you must delete the entity"
       );
     }
+
+    const subscription = await this.billingService.getSubscription(
+      user.workspace?.id
+    );
+
+    await this.analytics.trackWithContext({
+      properties: {
+        name: moduleAction.name,
+        operation: "delete",
+        planType: subscription.subscriptionPlan,
+      },
+      event: EnumEventType.InteractUserAction,
```

Comment:
Do you know this is never on an implication action here (e.g. use InteractAmplicationAction in some cases)?